src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala

```diff
@@ -262,6 +252,7 @@ trait WorkspaceApiService extends FireCloudRequestBuilding with FireCloudDirecti
               } ~
               path("acl") {
                 patch {
+                  // TODO CORE-382: can this be a passthrough?
```

Comment:
this isn't related to Data Library so I'm leaving it out of this PR; I'll look to create a separate PR for this